labgrid-web-client/src/app/resource/resource.component.ts

```diff
@@ -92,4 +103,20 @@ export class ResourceComponent implements OnInit {
 
         this.dataReady = true;
     }
+
+    private async readAcquiringUser(): Promise<void> {
+        const place = await this._ps.getPlace(this.resource.place);
+
+        if (place !== undefined) {
+            this.placeName = place.name;
+
+            if (place.acquired === 'labby/dummy') {
```

Comment:
The user's name should probably be stored in a variable, instead of being hard coded in, as in a future revision we might (we most certainly will) properly handle user credentials.

Reply:
Good point. Currently, all users are treated the same. But with credentials and authentication in the future, we need to distinguish between different user names.